workflow/java/src/main/java/io/dapr/quickstarts/workflows/WorkflowApp.java

```diff
@@ -0,0 +1,87 @@
+package io.dapr.quickstarts.workflows;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+import io.dapr.workflows.runtime.WorkflowRuntime;
+import io.dapr.workflows.runtime.WorkflowRuntimeBuilder;
+import io.dapr.workflows.client.DaprWorkflowClient;
+import io.dapr.workflows.client.WorkflowInstanceStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import io.dapr.quickstarts.workflows.models.*;
+import io.dapr.quickstarts.workflows.activities.*;
+
+@SpringBootApplication
+@RestController
+public class WorkflowApp {
+
+  @Autowired
+  private WorkflowRuntime workflowRuntime;
+
+  private static final Logger logger = LoggerFactory.getLogger(NotifyActivity.class);
+  private DaprWorkflowClient workflowClient;
+
+  public static void main(String[] args) {
+    SpringApplication.run(WorkflowApp.class, args);
+  }
+
+  @PostMapping("/workflow/start")
+  public ResponseEntity<String> startWorkflow(@RequestBody OrderPayload order) {
+    logger.info(String.format("Received request to start workflow for item: %s with quantity: %d", order.getItemName(),
```

Comment:
It's considered good form to use the logging library's built-in formatting:

```suggestion
    logger.info("Received request to start workflow for item: {} with quantity: {}", order.getItemName(),
```

That way log lines that won't be output (DEBUG lines in an INFO context) don't need to be formatted, which can be costly. (Every other log line is correct)

Reply:
great to know thx!